mappings/net/minecraft/entity/player/AbstractClientPlayer.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_40 net/minecraft/client/player/BaseClientPlayer
+CLASS net/minecraft/class_40 net/minecraft/entity/player/AbstractClientPlayer
```

Comment:
why are you moving it to the entity package?

Reply:
also BaseClientPlayer is fine, don't change names that are perfectly fine

Reply:
Entity package is where the actual Player entity is, and they should be in the same place imo. Can change it if you want.

Reply:
Also BaseClientPlayer was changed because Abstract seemed to be the code style used everywhere else.

Reply:
ah